### PR TITLE
Added Jenkinsfile

### PR DESCRIPTION
**Pull request by oshercc**
tested push, can inspect here :
http://10.46.10.2:8080/job/assisted-installer/job/osher%252Ftest/15/

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  environment { IMAGE = 'ocpmetal/assisted-installer'}
+  agent {
+    node {
+      label 'bm-inventory-subsystem'
+    }
+  }
+  stages {
+    stage('build') {
+      steps {
+        sh 'docker build . -f Dockerfile.assisted-installer-build -t ocpmetal/assisted-installer'
```
Comment (ronniel1) [inline Jenkinsfile:12]: Pleas use the Makefile

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
+pipeline {
+
+  environment { IMAGE = 'ocpmetal/assisted-installer'}
```
Comment (ronniel1) [inline Jenkinsfile:3]: Makefile uses the `INSTALLER` and `CONTROLLER` environment variable

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+              when {
+                  branch 'master'
+              }
+
+              steps {
+                  withCredentials([usernamePassword(credentialsId: 'ocpmetal_cred', passwordVariable: 'PASS', usernameVariable: 'USER')]) {
+                      sh '''docker login quay.io -u $USER -p $PASS'''
+                  }
+
+                  sh '''docker tag  ${IMAGE} quay.io/ocpmetal/assisted-installer:latest'''
```
Comment (ronniel1) [inline Jenkinsfile:27]: Also need to push the assisted-installer-controller image

File: Jenkinsfile (hunk 1 above)
Comment (oshercc) [reply Jenkinsfile:12]: I didn't use  the Make file since it uses skipper
if we use the docker file we are limited to running on the hosts

**Pull request merged**